dxr/plugins/clang/condense.py

```diff
@@ -283,10 +287,36 @@ def lines_from_csv(path):
     # This globbing is stupid but actually not that slow: a few tenths of a
     # second on a dir of 97K files in VirtualBox. That said, it does add up.
     paths = glob(join(folder, file_glob))
+
     return chain.from_iterable(lines_from_csv(p) for p in paths)
 
 
-def condense_file(csv_folder, file_path, overrides, overriddens, parents, children):
+def lines_from_csvs_with_path_digest(folder, csvdict, path_digest):
+    """Return an iterable of lines from all CSVs that 
+       start with the specified file path digest.
+
+    All lines are lists of strings.
+
+    :arg folder: The folder in which to look for CSVs
+    :arg csvdict: A dict of sha1 hex digest keys
+    :arg path_digest: A string produced by sha1(file_path).hexdigest()
+
+    """
+    def lines_from_csv(path):
```

Comment:
This looks like a copy-and-paste of the same inner function above; let's factor it out and not repeat ourselves.